OPT_mission.Altis/warehouse/functions/fnc_spawnvehicle.sqf

```diff
@@ -105,7 +105,12 @@ for "_i" from 0 to _spiralMaxPoints step 1 do
 		// update budget initialized by server!
 		[UNIT_NAME(_unit), UNIT_SIDE(_unit), _vecType, _unitCost, "-"] call EFUNC(common,updateBudget);
 		[] remoteExecCall [QFUNC(updateBudget), _unit, false];
-
+		
+		if ((_vec isKindOf "OPT_B_Heli_Light_01_F") or (_vec isKindOf "OPT_O_Heli_Light_01_F")) then 
```

Comment:
Ist das nicht im Mod behoben worden? Muss das in Mission und Mod entfernt werden?

Reply:
ja das kann nun aus der Mission wieder heraus